features/step_definitions/admin_policy_tagging_steps.rb

```diff
@@ -0,0 +1,8 @@
+Then(/^I can tag it to some policies$/) do
+  select policy_1["title"], from: 'Policies'
+  click_button 'Save'
+  publication = Publication.last
+
+  assert_path admin_publication_path(publication)
+  assert_equal [policy_1["content_id"]], publication.policy_content_ids
+end
```

Comment:
Could you push these capybara actions and checks down into helpers? e.g. https://github.com/alphagov/whitehall/blob/master/features/support/governments_helper.rb